PixArt/loader.py

```diff
@@ -23,10 +23,11 @@ def __init__(self, model_conf):
 	def model_type(self, state_dict, prefix=""):
 		return comfy.model_base.ModelType.EPS
 
-def load_pixart(model_path, model_conf):
+def load_pixart(model_path, model_conf, target_dtype: torch.dtype | None=None):

```

Comment:
I don't think adding typing is necessary considering the rest of the repo doesn't use any.